blockchain/consensus/deleg_cns/src/validation.rs

```diff
@@ -11,6 +11,18 @@ use fvm_shared::address::Address;
 
 use crate::DelegatedConsensusError;
 
+impl From<forest_chain::Error> for Box<DelegatedConsensusError> {
+    fn from(err: forest_chain::Error) -> Self {
+        Box::new(Into::into(err))
+    }
+}
+
+impl From<forest_state_manager::Error> for Box<DelegatedConsensusError> {
+    fn from(err: forest_state_manager::Error) -> Self {
+        Box::new(Into::into(err))
+    }
+}
+
```

Comment:
Maybe move these to `blockchain/consensus/deleg_cns/src/consensus.rs` where `DelegatedConsensusError` is defined (and where the other From impls are implicitly defined).

Reply:
Moved